packages/playground/src/api/functions/functionSchedulesApi.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 Cognite AS
+
+import { ItemsResponse } from '@cognite/sdk';
+import { BaseResourceAPI, CogniteInternalId } from '@cognite/sdk-core';
+import {
+  FunctionSchedule,
+  FunctionScheduleCreate,
+  FunctionSchedulesFilter,
+} from '../../types';
+
+export class FunctionSchedules extends BaseResourceAPI<FunctionSchedule> {
+  public retrieve = (ids: CogniteInternalId[], ignoreUnknownIds?: boolean) => {
+    return this.retrieveEndpoint(ids.map(id => ({ id })), {
+      params: ignoreUnknownIds,
+    });
+  };
+
+  public list = async (filter?: FunctionSchedulesFilter, limit?: number) => {
+    const response = await this.post<ItemsResponse<Function>>(
+      this.listPostUrl,
+      {
+        data: { filter, limit },
+      }
+    );
+    return response.data;
+  };
+
+  public create = async (items: FunctionScheduleCreate[]) => {
+    return this.createEndpoint(items);
+  };
+
+  public delete = async (ids: CogniteInternalId[]) => {
+    return this.deleteEndpoint(ids.map(id => ({ id })));
```

Comment:
why doing this mapping? why can't just take `IdEither[]`?

Reply:
(same for other places)

Reply:
Retrieve on function schedules only takes `id`, and not `externalId`. So we don't want the user to send `externalIds` when it is not supported. https://docs.cognite.com/api/playground/#operation/byidsFunctionSchedules